qiskit/providers/ibmq/api_v2/clients/account.py

```diff
@@ -123,30 +123,34 @@ def list_jobs_statuses(
         return self.client_api.jobs(limit=limit, skip=skip,
                                     extra_filter=extra_filter)
 
-    def job_submit(self, backend_name: str, qobj_dict: Dict[str, Any]) -> Dict[str, Any]:
+    def job_submit(self, backend_name: str, qobj_dict: Dict[str, Any],
+                   job_name: str = None) -> Dict[str, Any]:
```

Comment:
Pinging @luisg5 for recommendations on the type hints - `Optional`?

Reply:
Thanks Diego! Yes, to be consistent with other type hints already defined, it might be best to specify the type hint for `job_name` as an `Optional[str]`. This would go for other function declarations as well. Essentially, it means the same thing, but abides to PEP 484, which we are trying to follow.